astropy/convolution/tests/test_convolve_speeds.py

```diff
@@ -2,8 +2,6 @@
 
 import timeit
 
-import numpy as np  # pylint: disable=W0611
```

Comment:
Funky, explicitly disabled the warning, but why?

Reply:
This is odd.  W0611 is for unused imports.  I have no idea why someone would explicitly disable that warning instead of simply removing the import.

Reply:
#1445 mentions avoiding inline imports inside some functions.

Reply:
wow that's old!